plugin/core/sessions.py

```diff
@@ -1050,10 +1066,17 @@ def on_payload(self, payload: Dict[str, Any]) -> None:
                 exception_log("Error handling {}".format(typestr), err)
 
     def response_handler(self, response_id: int, response: Dict[str, Any]) -> Tuple[Optional[Callable], Any, bool]:
-        handler, error_handler = self._response_handlers.pop(response_id, (None, None))
-        if not handler:
+        request, handler, error_handler = self._response_handlers.pop(response_id, (None, None, None))
+        if not request:
             error = {"code": ErrorCode.InvalidParams, "message": "unknown response ID {}".format(response_id)}
             return (print_to_status_bar, error, True)
+        if request.view:
+            sv = self.session_view_for_view_async(request.view)
```

Comment:
These should also be done from sublime's async thread (currently this is in stdio read thread).
(Just a note)